library/src/main/kotlin/com/oddin/oddsfeedsdk/mq/entities/Outcome.kt

```diff
@@ -170,7 +172,7 @@ class OutcomeSettlementImpl(
     refId: Long?,
     marketData: MarketData,
     locale: Locale,
-    private val void: OFVoidFactor?,
+    private val void: Double?,
```

Comment:
I think, that this is not the best Kotlin approach how implement class with property voidFactor: VoidFactor? when input is Double?. I suggest to use secondary constructor https://kotlinlang.org/docs/classes.html#secondary-constructors and also probably it would be better if this class would be "data class" instead of "normal class".